runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -1560,6 +1565,7 @@ J9::ValuePropagation::isArrayCompTypeValueType(TR::VPConstraint *arrayConstraint
    //
    //   - Is no information available about the component type of the array?
    //     If not, assume it might be a value type.
+   //   - Is the component type definitely a identity type?
    //   - Is the component type definitely a value type?
    //   - Is the component type an array class (i.e., is this an array of
    //     arrays)?  If so, it's definitely not a value type.
```

Comment:
This comment can be removed if the check for `isClassArray` can also removed.

Reply:
The comment is removed in f926136